include_absolute.rb

```diff
@@ -147,7 +147,7 @@ def realpath_prefixed_with?(path, dir)
 
       # This method allows to modify the file content by inheriting from the class.
       def read_file(file, context)
-        File.read(file, file_read_opts(context))
+        File.read(file, **file_read_opts(context))
```

Comment:
@Soptq Thanks for the PR. Will this change break under Python 2?

Reply:
I don’t think so, it is a official code updating method mentioned at https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

Reply:
I just make a quick Jekyll 4.1 site build under Ruby 2.7 and it works perfectly. In terms of lower version, I think they are very likely to work as well, but I can’t confirm it as Jekyll 4.1 will raise exception if Ruby is under lower than 2.7 version.

Reply:
Thanks for confirmation, I'm merging it.